sumologic/resource_sumologic_cse_custom_entity_type.go

```diff
@@ -0,0 +1,129 @@
+package sumologic
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"log"
+)
+
+func resourceSumologicCSECustomEntityType() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSumologicCSECustomEntityTypeCreate,
+		Read:   resourceSumologicCSECustomEntityTypeRead,
+		Delete: resourceSumologicCSECustomEntityTypeDelete,
+		Update: resourceSumologicCSECustomEntityTypeUpdate,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: false,
```

Comment:
No need for `ForceNew` if it's false

Reply:
done under https://github.com/SumoLogic/terraform-provider-sumologic/pull/275/commits/6f2a00385fe4868446463043e99183b6d4b80bd7